src/stores/configStore.js

```diff
@@ -214,6 +214,7 @@ export class ConfigStore {
         value = value.split('%2C');
 
         value.forEach((item) => {
+          item = item.replace('%20', ' ');
```

Comment:
Why do this manually? Please use the standard methods (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/decodeURIComponent) or we will have to fix another edge case in the future